configure-gcloud-docker-gcloud-v101/action.yml

```diff
@@ -19,7 +19,9 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: google-github-actions/auth@v0
+    - id: 'auth'
+      name: 'Authenticate to Google Cloud'
+      uses: 'google-github-actions/auth@v2'
       with:
         credentials_json: ${{ inputs.gcloud-token }}
     - uses: google-github-actions/setup-gcloud@v1.0.1
```

Comment:
Just a minor comment that you can ignore: Should we also add `id` and `name` to this one `google-github-actions/setup-gcloud@v1.0.1` to stay consistent with the new change? Otherwise it looks good to me, thanks @goulter

Reply:
Good improvement, thanks @soufianerafik

Reply:
https://github.com/UWIT-IAM/actions/pull/28/commits/c4fe94f9e8b5d083c3fd5c29a3d07dd5d86d3b69